File: Python/Program/Shirt.py
from Clothing import Clothing

class Shirt(Clothing):
    
    # Private Attributes
    __color = ""
    __sleeveType = ""
    
    # Constructor
    def __init__(self, idProduct="", name="", brand="", price="", size="", material="", gender="", color="", sleeveType=""):
        super().__init__(idProduct, name, brand, price, size, material, gender)
        self.__color = color
        self.__sleeveType = sleeveType

    # Getter and Setter
    # Getter 
    def getColor(self):
        return self.__color

    def getSleeveType(self):
        return self.__sleeveType

    # Setter 
    def setColor(self, color):
        self.__color = color

    def setSleeveType(self, sleeveType):
        self.__sleeveType = sleeveType

File: PHP/Program/Shirt.php
<?php
require_once 'Clothing.php';

class Shirt extends Clothing {
    // Private Attributes
    private $color = "";
    private $sleeveType = "";
    
    // Constructor
    public function __construct($idProduct="", $name="", $brand="", $price="", $size="", $material="", $gender="", $color="", $sleeveType="") {
        parent::__construct($idProduct, $name, $brand, $price, $size, $material, $gender);
        $this->color = $color;
        $this->sleeveType = $sleeveType;
    }

    /*Getter & Setter*/
    // Getter
    public function getColor() {
        return $this->color;
    }

    public function getSleeveType() {
        return $this->sleeveType;
    }

    // Setter
    public function setColor($color) {
        $this->color = $color;
    }

    public function setSleeveType($sleeveType) {
        $this->sleeveType = $sleeveType;
    }
}

?>
